src/components/data/WdsTroubleshooter.js

```diff
@@ -0,0 +1,202 @@
+          (!!proxyUrl && proxyUrl !== 'unknown' ? checkIcon('success') : checkIcon('failure'))]),
+        td({ style: { fontWeight: 'bold' } }, ['Data Table app proxy url']),
+        // don't use the clippy() helper here so we can truncate the proxyUrl
+        td([
+          h(Fragment, [h(div, { style: _.merge({ width: '400px', float: 'left' }, Style.noWrapEllipsis) }, [proxyUrl]),
+            h(ClipboardButton, {
+              'aria-label': 'Copy proxy url to clipboard',
+              style: { marginLeft: '1rem' },
+              text: proxyUrl
+            })])
+        ])
+      ]),
+      tr([
+        td({ style: { fontWeight: 'bold' } }, [wdsResponsive == null ? checkIcon('running') :
+          (!!wdsResponsive && wdsResponsive !== 'unknown' ? checkIcon('success') : checkIcon('failure'))]),
+        td({ style: { fontWeight: 'bold' } }, ['Data Table app responding']),
+        td([h(Fragment, [wdsResponsive != null ? JSON.stringify(wdsResponsive) : ''])])
+      ]),
+      tr([
+        td({ style: { fontWeight: 'bold' } }, [version == null ? checkIcon('running') :
+          (!!version && version !== 'unknown' ? checkIcon('success') : checkIcon('failure'))]),
+        td({ style: { fontWeight: 'bold' } }, ['Data Table app version']),
+        td([clippy('WDS version', version)])
+      ]),
+      tr([
+        td({ style: { fontWeight: 'bold' } }, [wdsStatus == null ? checkIcon('running') :
+          (!!wdsStatus && wdsStatus !== 'unresponsive' && wdsStatus !== 'DOWN' ? checkIcon('success') : checkIcon('failure'))]),
+        td({ style: { fontWeight: 'bold' } }, ['Data Table app status']),
+        td([h(Fragment, [wdsStatus])])
+      ]),
+      tr([
+        td({ style: { fontWeight: 'bold' } }, [wdsDbStatus == null ? checkIcon('running') :
+          (!!wdsDbStatus && wdsDbStatus !== 'unknown' && wdsDbStatus !== 'DOWN' ? checkIcon('success') : checkIcon('failure'))]),
+        td({ style: { fontWeight: 'bold' } }, ['Data Table app DB status']),
+        td([h(Fragment, [wdsDbStatus])])
+      ]),
+      tr([
+        td({ style: { fontWeight: 'bold' } }, [wdsPingStatus == null ? checkIcon('running') :
+          (!!wdsPingStatus && wdsPingStatus !== 'unknown' && wdsPingStatus !== 'DOWN' ? checkIcon('success') : checkIcon('failure'))]),
+        td({ style: { fontWeight: 'bold' } }, ['Data Table app ping status']),
+        td([h(Fragment, [wdsPingStatus])])
+      ]),
+      tr([
+        td({ style: { fontWeight: 'bold' } }, [defaultInstanceExists == null ? checkIcon('running') :
+          (!!defaultInstanceExists && defaultInstanceExists !== 'unknown' ? checkIcon('success') : checkIcon('failure'))]),
+        td({ style: { fontWeight: 'bold' } }, ['Default instance exists']),
+        td([h(Fragment, [defaultInstanceExists])])
+      ]),
+    ]),
+    h(Fragment, {}, ['Please copy this information and email support@terra.bio to troubleshoot the error with your data tables.',
+      h(ClipboardButton, {
```

Comment:
I left a few of the copy icons in individual rows just because for me personally, it's convenient to just, say, copy the workspaceId alone.  Does it make sense to keep it for some of the longer/widely re-used values or should it just be copy-everything only?

Reply:
I think having the copy icon for (selected) individual rows is great!